tests/core/test_array.py

```diff
@@ -156,50 +151,47 @@ def test_trapezoid_product() -> None:
     np.testing.assert_allclose(s, 1.0)
 
 
-@pytest.mark.skipif(not HAVE_SCIPY, reason="test requires SciPy")
 def test_cumulative_trapezoid() -> None:
-    import scipy.integrate as spi
-
     # 1D f and x
```

Comment:
scipy just compares the result with the actual values - https://github.com/scipy/scipy/blob/1d57f315d9454446d333d37d19f2a44f4baf35c9/scipy/integrate/tests/test_quadrature.py#L183